R-package/R/lgb.train.R

```diff
@@ -18,11 +18,11 @@
 #'                   original datasets
 #' @param ... other parameters, see Parameters.rst for more information. A few key parameters:
 #'            \itemize{
-#'                \item{boosting}{Boosting type. \code{"gbdt"} or \code{"dart"}}
-#'                \item{num_leaves}{number of leaves in one tree. defaults to 127}
-#'                \item{max_depth}{Limit the max depth for tree model. This is used to deal with
+#'                \item{\code{boosting}: Boosting type. \code{"gbdt"} or \code{"dart"}}
+#'                \item{\code{num_leaves}: number of leaves in one tree. defaults to 127}
```

Comment:
From where this default value is taken from? If from cpp side, then it is not true anymore: https://lightgbm.readthedocs.io/en/latest/Parameters.html#num_leaves
```suggestion
#'                \item{\code{num_leaves}: Number of leaves in one tree. Defaults to 127.}
```